editor/plugins/script_editor_plugin.cpp

```diff
@@ -2097,13 +2098,35 @@ void ScriptEditor::_update_members_overview() {
 		functions.sort();
 	}
 
-	for (int i = 0; i < functions.size(); i++) {
-		String filter = filter_methods->get_text();
-		String name = functions[i].get_slicec(':', 0);
-		if (filter.is_empty() || filter.is_subsequence_ofn(name)) {
+	String filter = filter_methods->get_text();
+	if (filter.is_empty()) {
+		for (int i = 0; i < functions.size(); i++) {
+			String name = functions[i].get_slicec(':', 0);
 			members_overview->add_item(name);
 			members_overview->set_item_metadata(-1, functions[i].get_slicec(':', 1).to_int() - 1);
 		}
+	} else {
+		HashMap<String, int> name_to_line;
+		PackedStringArray search_names;
+
+		for (int i = 0; i < functions.size(); i++) {
+			String name = functions[i].get_slicec(':', 0);
+			int line = functions[i].get_slicec(':', 1).to_int() - 1;
+			name_to_line[name] = line;
+			search_names.append(name);
+		}
+
+		Vector<FuzzySearchResult> results;
+		FuzzySearch fuzzy;
+		fuzzy.set_query(filter);
```

Comment:
```suggestion
		fuzzy.set_query(filter);
		fuzzy.case_sensitive = false;
```
This should also not be case sensitive, since it wasn't before.

Reply:
Done, but blocked by https://github.com/godotengine/godot/pull/105900